lua/entities/starfall_processor/cl_init.lua

```diff
@@ -185,3 +185,61 @@ net.Receive("starfall_processor_used", function(len)
 	end
 end)
 
+-- Returns ply or nil
+local function getPlayerBySteamID( idStr )
+	if not idStr then
+		LocalPlayer():PrintMessage( HUD_PRINTCONSOLE, "Missing SteamID\n" )
+		return
+	end
+
+	-- Check if SteamID64
+	if tonumber( idStr ) then
+		idStr = util.SteamIDFrom64( idStr )
+	end
+
+	local ply = player.GetBySteamID( idStr )
+	if ply then
+		return ply
+	else
+		LocalPlayer():PrintMessage( HUD_PRINTCONSOLE, "Player Not Found\n" )
+		return nil
+	end
+end
+
+-- Local function to handle autocomplete for the terminate commands
+local function terminateAutoComplete( cmd, stringArgs )
+	local examples = {}
+	for _, ply in pairs( player.GetHumans() ) do
+		table.insert( examples, cmd .. " " .. "\"" .. ply:SteamID() .. "\" " .. "\"" .. ply:Nick() .. "\"" )
+	end
+
+	return examples
+end
+
+---Terminates a user's starfall chips clientside
+---Supports SteamID and SteamID64
+concommand.Add( "sf_kill_cl", function( executor, cmd, args )
+	local ply = getPlayerBySteamID( args[1] )
+	if not ply then return end
+
+	for instance, _ in pairs( SF.playerInstances[ply] ) do
+		instance:Error( { message = "Killed by user", traceback = "" } )
+	end
+end, terminateAutoComplete, "Terminates a user's starfall chips clientside." )
+
+---Terminates a user's starfall chips
+---This command is admin only
+concommand.Add( "sf_kill", function( executor, cmd, args )
+	if not executor:IsAdmin() then return end
+
+	local ply = getPlayerBySteamID( args[1] )
+	if not ply then return end
+
+	if SF.playerInstances[ply] then
+		for instance, _ in pairs( SF.playerInstances[ply] ) do
+			net.Start( "starfall_processor_kill" )
+			net.WriteEntity( instance.entity )
+			net.SendToServer()
+		end
+	end
+end, terminateAutoComplete, "SuperAdmin Only. Terminate a user's starfall chips." )
```

Comment:
One more thing, this help text should say admin only.

Reply:
Oh, yeah my bad. I'll fix that 👍